user_docs/en/changes.t2t

```diff
@@ -106,10 +107,10 @@ Same as pressing ``NVDA+k`` twice, but may be more useful for braille users. (#1
 
 == Changes for Developers ==
 Note: this is an Add-on API compatibility breaking release.
-Add-ons will need to be re-tested and have their manifest updated.
+Add-ons will need to be re-tested and have their manifests updated.

```

Comment:
I think this change has been discussed before, and either sentence is fine. The current grammar can be read as "the [add-ons should] have their manifest updated".

If this new string is preferred, we need to update a template elsewhere